src/status_im/utils/utils.cljs

```diff
@@ -62,11 +59,10 @@
 
 ;; same as re-frame dispatch-later but using background timer for long
 ;; running timeouts
-(re-frame/reg-fx
- :utils/dispatch-later
- (fn [params]
-   (doseq [{:keys [ms dispatch]} params]
-     (set-timeout #(re-frame/dispatch dispatch) ms))))
+(re-frame/reg-fx :utils/dispatch-later
+                 (fn [params]
```

Comment:
this change is not needed. It was correctly formatted code before and aligned with clojure styleguides.

Reply:
When I first attempted to commit *my* changes the pre-check rejected the commit because of formatting errors. At the time I did a sanity check by switching back to `develop` to make sure my code was not involved; checking out a new branch `fixtest; making an innocuous textual change; and then trying to commit. The formatting check failed the existing code, and making these changes cleared the failure. Would you like me to repeat the exercise?

Reply:
btw, I was fine with the original code, I was just reacting to the pre-commit rejection.

Reply:
weird, @kennytilton is it possible that you have some auto-formatting in the editor and they apply on save to the file?

Reply:
@kennytilton yes, please,  `lein cljfmt check src/status_im/utils/utils.cljs` does not show any formatting errors for me. Even after i've added some dummy new code to it just to have any new changes inside that file.

Also, we have global check for every PR https://github.com/status-im/status-react/blob/develop/Jenkinsfile#L44 that check all files so those formatting errors should have been detected before it they really were there.

Reply:
Let me look into this. I will use `nano` or `sublime` and repeat the exercise. I edit with IntelliJ/Cursive and frequently use the "reformat all option" during development, maybe I did that unconsciously even during my sanity check.

Reply:
Thanks for checking that @dmitryn.I will revert and see if I can get it committed.

The one difference I see is that I am not running `lein cljmt check` myself, the commit pre-check is doing it. But looking at the :pre-commit option in project.clj I do not see how that would explain anything.

Reply:
OK, second time through no issues. It must have been a habitual global reformat. I will have to explore eventually why my defaults are displeasing `lein cljfmt`. I just pushed all the changes to my branch, studying up now on how to propagate that for review. I'd like to complete this iteration before looking at the latest input on a new approach.